# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/res/prim_shared.glsl
# Commit: ee0de0eac18d
# Full Hash: ee0de0eac18d1ba825467f6629ded5e613c56627
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-18 04:07:40
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 1 - Remove PrevPassAlpha and PrevPassColor r=nical
#   
#   Remove usage of the implicit prev pass alpha and color texture
#   samplers from batching / renderer / shader code. They are replaced
#   by explicit references to the texture ID for the source task.
# ==============================================================================

diff -r ecf2ab52eb03 -r ee0de0eac18d gfx/wr/webrender/res/prim_shared.glsl
--- a/gfx/wr/webrender/res/prim_shared.glsl	Thu Dec 17 22:10:02 2020 +0000
+++ b/gfx/wr/webrender/res/prim_shared.glsl	Thu Dec 17 22:26:08 2020 +0000
@@ -15,8 +15,6 @@
 #define RASTER_LOCAL            0
 #define RASTER_SCREEN           1
 
-uniform sampler2DArray sPrevPassAlpha;
-uniform sampler2DArray sPrevPassColor;
 uniform sampler2DArray sClipMask;
 
 vec2 clamp_rect(vec2 pt, RectWithSize rect) {